multinet/user.py

```diff
@@ -1,163 +1,204 @@
 """User data and functions."""
+from __future__ import annotations  # noqa: T484
 
-import dataclasses
+import json
+from dataclasses import dataclass
 from uuid import uuid4
-from arango.collection import StandardCollection
-from arango.cursor import Cursor
+from copy import copy
 from dacite import from_dict
-from flask import session
+from flask import session as flask_session
 
-from multinet.db import db, read_only_db, _run_aql_query
-from multinet.errors import InternalServerError
-from multinet.auth.types import (
-    GoogleUserInfo,
-    MultinetInfo,
-    UserInfo,
-    User,
-    FilteredUser,
-)
+from multinet.db import user_collection, system_db, _run_aql_query
 
-from typing import Optional, Dict
+from typing import Optional, Dict, Generator, Any
 
 MULTINET_COOKIE = "multinet-token"
 
 
-def user_collection() -> StandardCollection:
-    """Return the collection that contains user documents."""
-    sysdb = db("_system")
+@dataclass
+class MultinetInfo:
```

Comment:
I feel like this needs a much better class name.

Why can't this information be included in the `UserInfo` class directly?

Reply:
`UserInfo` represents the data that can be used to create a new User, while `MultinetInfo` represents the additional data that is added by our system, and that we use. This may grow, which is why having a separate dataclass to represent it is beneficial.

Reply:
Makes sense. I still think we should come up with a better name than "MultinetInfo". But I see that you're simply updating what was there before, so I've filed #461 and will resolve this conversation.